deployment-aws/puppet/mevapi/templates/cloudwatch_config.json.epp

```diff
@@ -0,0 +1,95 @@
+{
+	"agent": {
+		"metrics_collection_interval": 60
+	},
+	"logs": {
+		"logs_collected": {
+			"files": {
+				"collect_list": [
+					{
+						"file_path": "<%= $mevapi::log_dir %>/gunicorn.log",
+						"log_group_name": "<%= $mevapi::cloudwatch_log_group %>",
+						"log_stream_name": "{instance_id}",
+						"retention_in_days": 14
```

Comment:
Should we make it 15 to match the metrics retention?

Reply:
makes sense.